pcs/prop.py

```diff
@@ -94,11 +85,8 @@ def unset_property(lib, argv, modifiers):
 
 
 def list_property_deprecated(lib, argv, modifiers):
-    deprecation_warning(
-        "This command is deprecated and will be removed. "
-        "Please use 'pcs property config' instead."
-    )
-    return list_property(lib, argv, modifiers)
+    del lib, argv, modifiers
+    raise_command_replaced(["pcs property config"], pcs_version="0.11")
```

Comment:
This is actually meant to be deprecated in pcs-0.11. Please, revert your changes.